hasura/graphql-engine
# evaluate dependencies for column operators in permissions (close #2054)

### Description



### Affected components 


- Server
- Tests


### Related Issues



close #2054 

### Solution and Design


Fetch dependant columns from `OpExpG a` for column comparison operators.  

### Steps to test and verify


Create permission with column operators and try to delete a column involved in permission rule through the console or `run_sql` query. This should throw a dependency error.
